dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -96,9 +96,11 @@ dependent_startup_wait_for=rsyslogd:running
 {% if not skip_xcvrd %}
 [program:xcvrd]
 {% if delay_xcvrd %}
-command=bash -c "sleep 30 && {% if API_VERSION == 3 and 'xcvrd' not in python2_daemons %}python3 {% else %} python2 {% endif %}/usr/local/bin/xcvrd"
+command={% if skip_xcvrd_cmis_mgr %} bash -c "sleep 30 && python3 /usr/local/bin/xcvrd --skip_cmis_mgr" {% else %} bash -c "sleep 30 && python3 /usr/local/bin/xcvrd" {% endif %}
+
```

Comment:
@prgeor  
skip_xcvrd_cmis_mgr is set in platform_daemon_control.json file and in turn used by supervisord to spawn xcvrd with --skip_cmis_mgr as an arg. 
So, better to make the variable name same at both places (producer/setter and consumer)